tests/fixtures/models/store.py

```diff
@@ -15,14 +16,174 @@
 
 import pytest
 
+from ...utils import update_store, create_store
+
+
+DEFAULT_STORE_UNITS_1 = [("Unit 1", "Unit 1"),
+                         ("Unit 2", "Unit 2")]
+
+DEFAULT_STORE_UNITS_2 = [("Unit 3", "Unit 3"),
```

Comment:
For lists/dicts with many items, always [break immediately after the opening brace](http://docs.translatehouse.org/projects/translate-toolkit/en/latest/developers/styleguide.html#continuing-long-statements).